src/main/java/winstone/WinstoneSession.java

```diff
@@ -9,5 +9,9 @@
     /**
      * Name of the cookie that stores HTTP session ID.
      */
-    public final static String SESSION_COOKIE_NAME = "JSESSIONID";
+    // @SuppressFBWarnings(value = "MS_SHOULD_BE_FINAL", justification = "mutated by extras-executable-war")
```

Comment:
#69 did not add the annotations JAR as a dep and I did not care to deal with that here. Anyway SB errors are not currently fatal.